### PR TITLE
Added savedJobs component 

### PR DESCRIPTION
**Pull request by hrahul2605**
PR for the issue #922 
#### add a savedJobs component under Organisms #922

### CODE REVIEW

**Comment by Dishebh** [comment]
@hrahul2605 Please provide a suitable PR description, like the issue you're fixing with this PR :)

**Comment by hrahul2605** [comment]
> @hrahul2605 Please provide a suitable PR description, like the issue you're fixing with this PR :)

I have updated the description with the issue. Please check now.

**Comment by Dishebh** [comment]
Add a `Heading` here as well, as you did in the `messageSettings` component.

**Comment by hrahul2605** [comment]
> Add a `Heading` here as well, as you did in the `messageSettings` component.

Done. I also modified the component a little. Have a look. :smile:

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Assets/ignitus-Icons/moreHorizontalIcon/index.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        y="-12"
+        width="50"
+        height="49"
+      >
+        <rect x="-13" y="-11.5957" width="50" height="47.8723" fill="white" />
+      </mask>
+      <g mask="url(#mask1)" />
+    </g>
+  </svg>
+);
```
Comment (Dishebh) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Assets/ignitus-Icons/moreHorizontalIcon/index.tsx:45]: See how other `ignitus-Icons` are implemented and try constructing this in a similar fashion :) (I don't think that the use of `mask` tag is necessary for this implementation)

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-SavedJobsCard/Components/index.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        <img src={avatar} alt="avatar" />
+      </S.ImageContainer>
+      <S.DetailContainer>
+        <S.JobTitle>{title}</S.JobTitle>
+        <S.JobDesc>
+          {poster} | {location}
+        </S.JobDesc>
+        <S.JobUpdate>Updated {lastUpdated}</S.JobUpdate>
+      </S.DetailContainer>
+      <S.RightItemsContainer style={{ right: isDraft ? '3%' : '8%' }}>
```
Comment (Dishebh) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-SavedJobsCard/Components/index.tsx:30]: Avoid inline styling

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-SavedJobsCard/Components/index.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        <S.MoreContainer>
+          <S.IconContainer onClick={() => setOpen(!open)}>
+            <S.Icon name={AppIcon.MoreHorizontalIcon} />
+          </S.IconContainer>
+          <DropdownMenu setOpen={setOpen} open={open} jobOptions={jobOptions} />
+        </S.MoreContainer>
+        {isDraft ? (
+          <Button size="large" category="secondary" style={{ marginLeft: 25 }}>
+            Publish
```
Comment (Dishebh) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-SavedJobsCard/Components/index.tsx:39]: Here as well

File: src/ignitus-UserInterfaceBook/Components/Organisms/interfaceSavedJobs/constants.ts [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    poster: 'Internship poster',
+    title: 'Internship Title',
+    avatar:
+      'https://storage.googleapis.com/ignitus_assets/ig-avatars/eugene.png',
+  },
+];
+
+const demoFucntion: Function = e => {
+  alert(`Function Call on ${e.title}`);
+};
```
Comment (Dishebh) [inline src/ignitus-UserInterfaceBook/Components/Organisms/interfaceSavedJobs/constants.ts:47]: Remove the alert function call for now

**Comment by Dishebh** [comment]
Avoid inline styling everywhere in the code :)

**Comment by Dishebh** [comment]
@hrahul2605 great work up till now! There's one small issue though. The dropdown menu remains open even when you click outside the icon. Can you fix this up?

P.S You can refer [here](https://stackoverflow.com/questions/32553158/detect-click-outside-react-component) if you wish, or you can take help from somewhere else :)

**Comment by hrahul2605** [comment]
> @hrahul2605 great work up till now! There's one small issue though. The dropdown menu remains open even when you click outside the icon. Can you fix this up?
> 
> P.S You can refer [here](https://stackoverflow.com/questions/32553158/detect-click-outside-react-component) if you wish, or you can take help from somewhere else :)

Okay, I'll look into that

**Comment by hrahul2605** [comment]
Hey @Dishebh, fixed the issue, please check.

**Review by Dishebh — APPROVED** [review]
Implementation-wise LGTM

Design review still pending - @Paarmita @divyanshu-rawat

**Review by divyanshu-rawat — APPROVED** [review]
Awesome @hrahul2605 👍

**Pull request merged**